drivers/segger/console_rtt.c

```diff
@@ -0,0 +1,127 @@
+/****************************************************************************
+ * drivers/segger/console_rtt.c
```

Comment:
console->serial

Reply:
Maybe you want to unify all RTT channels into one device registration interface.

But channel 0 has some special features, it can be used without any initialization, and other channels need to be configured (corresponding to the semantics of open)
I plan to use channel 0 as a console and create a new /dev/ttyRTTn device for other channels

So I think it would be better to separate the two, what do you think?

[https://wiki.segger.com/RTT#TELNET_channel_of_J-Link_software](https://wiki.segger.com/RTT#TELNET_channel_of_J-Link_software:~:text=RTT%20Channel%200%20is%20somewhat%20special%3A)